spec/routing/routes_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+describe 'routes', type: :routing do
```

Comment:
The argument for adding it is that devise might decide to bring it back for whatever reason. So because we're dependent on a third party gem, we want to be extra sure. That being said I'm not particularly enamoured to it.